internal/expr/parser.go

```diff
@@ -423,7 +450,8 @@ func (p *Parser) parseList(parseFn func(p *Parser) (fullName, bool, error)) ([]f
 		return nil, false, nil
 	}
 
-	parenPos := p.pos
+	// Error points to first parentheses skipped above.
+	parenCol := p.colNum() - 1
```

Comment:
This could maybe be called `openingParenCol` to make it more explicit.

Also, should you also worry about the line number here? It is possible to have a multi line comment in a list, also `skipBlanks` skips new lines.

Reply:
You are right about both, added a test and corrected the second one.